client/my-sites/themes/events/theme-showcase-tracks.ts

```diff
@@ -51,7 +51,7 @@ const getThemeShowcaseEventRecorder = (
 				is_active_theme: isActiveTheme( themeId ),
 				style_variation: styleVariation,
 				results_rank: adjustedPosition,
-				results: themes.map( property( 'id' ) ).join(),
+				results: themes?.map( property( 'id' ) ).join() ?? [],
```

Comment:
Non-blocking, i'm ok either way: Is this a good opportunity to remove lodash's `property` ?

Reply:
Sure

Reply:
Done! thanks!